legacy-content-import/src/main/scala/legacycontentimport/Main.scala

```diff
@@ -7,6 +7,7 @@ object Main extends App {
 
   val articles = Article.fromCapiHelpSection(capiDomain, capiKey)
   articles foreach { article =>
-    println(s"""${article.url},"${article.title.replace("\"", "'")}",${article.publicationDate}""")
+    println(s"""${article.url},"${article.title.replace("\"", "'")}",${article.publicationDate},"${article.keywords
```

Comment:
Formatting seems confusing here. Note how `"${article.keywords.mkString(", ")}"` is broken across two lines.

Reply:
Refactored it a bit now so clearer to make out fields.